assets/src/edit-story/components/richText/karma/singleSelection.karma.js

```diff
@@ -330,4 +330,36 @@ describe('Styling single text field', () => {
       expect(actual).toBe(expected);
     });
   });
+
+  describe('CUJ: Creator Can Style Text: When a font without weight support replaces font, default font weight to closest match', () => {
+    it('should replace Bold Roboto with normal weight font for Abel since Bold is not an option', async () => {
+      const {
+        fontWeight,
+      } = data.fixture.editor.inspector.designPanel.textStyle;
+      expect(fontWeight.value).toBe('Regular');
+
+      await data.fixture.events.click(fontWeight.select);
+      await data.fixture.events.click(fontWeight.option('Bold'));
+      expect(fontWeight.value).toBe('Bold');
+
+      const input = await data.fixture.screen.getByLabelText(
```

Comment:
Could we make this part of the `fixture.editor.inspector.designPanel` instead?

Reply:
Thanks for mentioning this, I'm new to the editor (mostly have been in the dashboard) and I missed that this was something to add to. 

I have added fontFamily to the karma fixture in this commit:  https://github.com/google/web-stories-wp/pull/4734/commits/86a185c752315bebc2f9e72aed2cdbf197c8bdff 

However we need to flesh out another `containers/common` for the font family element since it isn't just a dropdown it's also a search when it's active. And this, to me, feels like it should be it's own update - plus then we can grab all of the references to the font family in tests and update them within one focused PR. Thoughts? 

if this is something straight forward and I've just missed it, lemme know - happy to keep going, just also want to stay on target and actually fix this bug.